plugins/legacy-plugin-chart-partition/test/optionDescirption.test.jsx

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
filename typo. Also please use Uppercase for the first character to match source file name

`O`ptionDes`crip`tion

Reply:
Wow, eagle eyes award goes to Krist! Can't believe I didn't see that.